WcaOnRails/app/views/competitions/_index_competitions_list.html.erb

```diff
@@ -50,4 +43,35 @@
       });
     });
   </script>
+<% else %>
+  <% if current_user&.can_see_admin_competitions? %>
+    <% if competitions.empty? %>
+      <%= render 'no_competitions_found' %>
+    <% else %>
+      <% past_competitions, upcoming_competitions = competitions.partition(&:is_over?) %>
+      <% recent_competitions, distant_competitions = past_competitions.partition { |c| (Date.today - c.end_date).to_i < 30 } %>
```

Comment:
We must have this number 30 somewhere else by now! Didn't you just switch it to 28, btw?